resolver/rcmd.py

```diff
@@ -1,57 +1,47 @@
 # -*- coding: utf-8 -*-
-
-from sceptre.resolvers import Resolver
-import os
 import subprocess
+from typing import Any
 
-
-def build_expression(args, profile):
-    '''
-    Build the command that will get executed. The expression always contains
-    an AWS profile. By default, the profile that the command is executed with
-    is the one that is used to run sceptre. The user can override that profile
-    by setting an alternate profile in the `profile` parameter.
-    :param args: input arguments
-    :param profile: the AWS profile to execute with
-    :return: the expression to execute
-    '''
-    if not args:
-        raise ValueError("Missing the command to execute")
-
-    if not profile:
-        profile = os.environ.get("AWS_PROFILE")
-
-    expression = args
-    if isinstance(expression, dict):
-        try:
-            expression = args['command']
-            if not expression:
-                raise ValueError("Missing the command to execute")
-        except KeyError:
-            raise KeyError("Missing the command to execute")
```

Comment:
these checks seems to be missing in the refactor,  any reason why?

Reply:
These checks are a bit silly in that they try to get the command and if there isn't one, they blow up with a Key Error. But catching a KeyError just to re-raise one seems silly to me. The KeyError will be raised here: https://github.com/Sceptre/sceptre-resolver-cmd/pull/12/files#diff-410020875feef0c3200bda5f50913cc5491eaccd8683ea95670cc5af00790ef3R41

I suppose that I didn't keep the check for a falsey command. I suppose I could implement that. 

Would you feel more comfortable if I emulated the falsey command check and maybe wrapped the KeyError message change?

Reply:
Ok, I've implemented checks to make sure a command is passed, no matter if it's as a string or in a dict.